packages/presets/src/fragments/bi-directional-link/bi-directional-link-panel.ts

```diff
@@ -201,16 +201,13 @@ export class BiDirectionalLinkPanel extends WithDisposable(LitElement) {
   }
 
   private get _docPageElement() {
-    const affineDocEditor = this.closest('affine-doc-editor');
-    assertExists(affineDocEditor);
-    const docPageElement = affineDocEditor.querySelector('affine-doc-page');
+    const docPageElement = this.parentElement?.querySelector('affine-doc-page');
```

Comment:
The parent element is not necessarily the one that wraps both `affine-doc-page` & `bi-directional-link`

Reply:
I think you can add a new property to let the upstream to pass in an doc-page element instead

Reply:
Alright, that sounds reasonable.

Reply:
done